### PR TITLE
Add resume tests

### LINKED ISSUE #196: Add tests for the `resume` feature
Need to add some tests for the resume feature. It includes generating a partial MAPS structure and add it to the testing data.

### LINKED ISSUE #274: Resume is not working when using multiple splits
The command `clinicadl resume MAPS_DIR` is not working when trying to resume a work using multiple splits. 

I get a `ValueError` from type:

```
 File "/.../clinicadl/utils/maps_manager/maps_manager.py", line 150, in resume
    f"Splits {missing_splits} were not initialized. "
ValueError: Splits [0, 1, 3, 4] were not initialized. Please try train command on these splits and resume only others.
```

Seems to be a bug when looking for stopped jobs in the maps folder. See the following file: 
```
clinicadl/train/resume.py
```

### PR DESCRIPTION
**Pull request by 14thibea**
Closes #196 and closes #274.
4 tests are performed
- `stopped_1`: only `maps.json` is present.
- `stopped_2`: `split-0` is created but no epoch was run yet.
- `stopped_3`: `split-0` is created and 2/5 epochs are present.
- `stopped_4`: `split-0` ended but `split-1` must be resumed.

### CODE REVIEW

**Comment by mdiazmel** [comment]
Hi @14thibea ,

I propose in the PR #275 a quickly fix for the resume bug (there is a problem when looking for the current splits). It seems that you also address this bug in this PR (with a more robust solution) so I will close the other one.

File: Jenkinsfile [hunk 1]
```diff
@@ -287,6 +287,35 @@ pipeline {
...
+                        --disable-warnings \
+                        test_resume.py
+                     rm -r data/stopped_jobs
+                     conda deactivate
+                     '''
+                }
+                post {
+                  always {
+                    junit 'tests/test-reports/test_resume_report.xml'
+                    sh 'rm -rf $WORKSPACE/tests/data/dataset'
```
Comment (mdiazmel) [inline Jenkinsfile:315]: Add a line to remove the `stopped_jobs` folder when tests are finished:
```
sh 'rm $WORKSPACE/tests/data/stopped_jobs'
```

File: clinicadl/train/resume.py [hunk 2]
```diff
@@ -53,14 +53,13 @@ def automatic_resume(model_path, verbose=0):
         if split not in finished_splits and split not in stopped_splits
     ]
 
-    logger.info(f"List of finished splits {finished_splits}")
-    logger.info(f"List of stopped splits {stopped_splits}")
-    logger.info(f"List of absent splits {absent_splits}")
     # To ensure retro-compatibility with random search
-    logger.info(
+    print(
```
Comment (mdiazmel) [inline clinicadl/train/resume.py]: Why this message is printed in the stdout and is not managed by the logger?

**Review by mdiazmel — COMMENTED** [review]
Thanks @14thibea !

I added a couple of comments to address. Otherwise seems ok, the test for resume seems to cover most of the possible cases.

File: clinicadl/train/resume.py (hunk 2 above)
Comment (14thibea) [reply clinicadl/train/resume.py]: Verbose is not working so I printed to know what was wrong...
I opened #276 to talk about it.

File: Jenkinsfile (hunk 1 above)
Comment (14thibea) [reply Jenkinsfile:315]: Added in https://github.com/aramis-lab/clinicadl/pull/272/commits/60fb8c1c3a5d800eff6872de413e76a33655d5a4

File: clinicadl/train/resume.py (hunk 2 above)
Comment (14thibea) [reply clinicadl/train/resume.py]: Changed in https://github.com/aramis-lab/clinicadl/pull/272/commits/60fb8c1c3a5d800eff6872de413e76a33655d5a4

Comment (mdiazmel) [reply clinicadl/train/resume.py]: > Verbose is not working so I printed to know what was wrong...
> I opened #276 to talk about it.

Ok, thanks !

**Review by mdiazmel — APPROVED** [review]
LGTM!

I'll merge this one!

**Pull request merged**